extensions/gluonTS/src/main/java/ai/djl/gluonTS/transform/Utils.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Let's check in those class when needed

Reply:
Please avoid use git merge in future, it's hard to read history

Reply:
And I suggest you break your changes into smaller pieces if possible

Reply:
Sorry for that, I wondering is there any tutorial or cmd for me to split the changes into pieces? Cuz, im not familiar with git

Reply:
> 

Thanks! I got it.